README.md

```diff
@@ -172,6 +173,31 @@ If you want to disable a service, you can simply remove the container for that s
   - Ensure your SMTP instance reachable by the OpenWISP containers.
   - Change the [email configuration variables](docs/ENV.md) to point to your instances.
 
+### Customizing uWSGI configuration
+
+By default, you can only configure [`processes`, `threads` and `listen`
+settings of uWSGI using environment variables](docs/ENV.md#uWSGI).
+If you want to configure more uWSGI settings, you can supply your uWSGI
+configuration by following these steps:
+
+1. Create the uWSGI configuration file in the same directory as `docker-compose.yml`.
+   For the sake of this example, let's assume the filename is `custom_uwsgi.ini`.
+2. In `dashboard` and `api` services of `docker-compose.yml`, add volumes as following
+
+```yml
+  services:
+    dashboard:
+      ... # other configuration
+      volumes:
+        ... # other volumes
+        - ${PWD}/custom_uwsgi.ini:/opt/openwisp/uwsgi.ini:ro
```

Comment:
Now that the changes in the settings PR (#192) are merged, we have directory for customizations, so I tried the following, which works:

```
${PWD}/customization/configuration/uwsgi.ini:/opt/openwisp/uwsgi.ini
``` 

I think we should recommend to concentrate all the customization in this directory, for 3 reasons:

- consistency
- the files in this directory are ignored by git
- we want to avoid recommending users to scatter configuration files in different places

What do you think?